demo/index.md

```diff
@@ -0,0 +1,28 @@
+---
+layout: demo
+---
+
+<div style="margin: 0px 1em 2em 1em;">
+On this page you can play with a live demo of the Cryostat Web UI. This runs entirely in your browser and does not access any real Cryostat backend server instances.
+Core JFR functionality is not available in this stubbed out demo, but you can use this to get a sense of what features Cryostat offers and what it looks like.
+</div>
+
+<div style="position: relative;">
+  <div
+    style="position: absolute; left: 0; top: 0; width: 100%; height: 100%; align-items: center; display: flex; justify-content: center">
+      <div id="loading" aria-label="Loading demo" class="linear-dot-spinner"></div>
+  </div>
+  <iframe
```

Comment:
Since cards might have white background, this could look pretty odd as iframe could appear blending into the outside.

![image](https://github.com/cryostatio/cryostatio.github.io/assets/68053619/c160106e-5c1d-4e26-afc6-ec5bf4d7725e)

How about adding a border (below also includes formatting to scss that u can ignore)?

```diff
diff --git a/_sass/core/demo.scss b/_sass/core/demo.scss
index 51127d0..6053f1e 100644
--- a/_sass/core/demo.scss
+++ b/_sass/core/demo.scss
@@ -1,9 +1,23 @@
-.linear-dot-spinner{
+.linear-dot-spinner {
     width: 2em;
     aspect-ratio: 4;
-    background: radial-gradient(circle closest-side, #73BCF7 90%,#0000) 0/calc(100%/3) 100% space;
+    background: radial-gradient(circle closest-side, #73BCF7 90%, #0000) 0/calc(100%/3) 100% space;
     clip-path: inset(0 100% 0 0);
     animation: linear-dot-spinner-animate 1s steps(4) infinite;
 }
 
-@keyframes linear-dot-spinner-animate {to {clip-path: inset(0 -34% 0 0)}}
+@keyframes linear-dot-spinner-animate {
+    to {
+        clip-path: inset(0 -34% 0 0)
+    }
+}
+
+.demo-iframe {
+    opacity: 0;
+    box-sizing: border-box;
+    width: 100%;
+    height: 50em;
+    scale: 0.98;
+    border: 1px solid;
+    border-radius: 4px;
+}
diff --git a/demo/index.md b/demo/index.md
index cd95800..b6fd99d 100644
--- a/demo/index.md
+++ b/demo/index.md
@@ -9,13 +9,14 @@ Core JFR functionality is not available in this stubbed out demo, but you can us
 
 <div style="position: relative;">
   <div
+    id="loading"
     style="position: absolute; left: 0; top: 0; width: 100%; height: 100%; align-items: center; display: flex; justify-content: center">
-      <div id="loading" aria-label="Loading demo" class="linear-dot-spinner"></div>
+      <div aria-label="Loading demo" class="linear-dot-spinner"></div>
   </div>
   <iframe
     id="demo-frame"
     src="{{ "/assets/demo/site/index.html" | relative_url }}"
-    style="opacity: 0; max-width: initial !important; width: 99%; height: 54em; margin: 0px 10px 0px 10px; border: none">
+    class="demo-iframe">
   </iframe>
 </div>
 <script>
```

![image](https://github.com/cryostatio/cryostatio.github.io/assets/68053619/95b1c088-b1fc-409e-a65b-daa4b89330b5)